content/en/docs/guides/custom-load-balancing.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Custom Load Balancing Policies
+description: >-
+  Explains custom load balancing policies can help optimize load balancing
+  under unique circumstances.
+---
+
+### Overview
+
+One of the key features of gRPC is load balancing, which allows requests from
+clients to be distributed across multiple servers. This helps prevent any
+one server from becoming overloaded and allows the system to scale up by adding
+more servers.
+
+A gRPC load balancing policy is given a list of server IP addresses by the name
+resolver. The policy is responsible for maintaining connections (subchannels)
+to the servers and picking a connection to use when an RPC is sent.
+
+### Implementing Your Own Policy
+
+By default the `pick_first` policy will be used. This policy actually does no
+load balancing but just uses the first address it gets from the name resolver.
+By updating the gRPC service config you can also switch to using `round_robin`
+that rotates through all the addresses it gets. But if the built-in load
+balancing policies do not meet your needs (the exact set available vary 
+between languages) you can also implement you own custom policy.
+
+This involves implementing a load balancer interface in the language you are
+using. At a high level, you will have to:
+
+- Register your implementation in the load balancer registry so that it can
+be referred to from the service config
+- Define the JSON configuration object of your implementation. This allows your
```

Comment:
The definition of this configuration object is sort of virtual. I'd probably emphasize "_parsing_ the JSON configuration object of your implementation."